elastic/kibana
# [Investigations] - Add unified components to eql tab

## Summary

This PR introduces the unified table and field list to the correlations tab as seen in the snapshots below.

**As of this PR, items that are not working**
1. Table row height controls
2. Expandable flyout integration
3. Leading cell actions (pinning, notes, row actions, analyzer, session view)

**Changes in this PR:**

Sequence Highlighting:

<img width="1715" alt="image" src="https://github.com/elastic/kibana/assets/17211684/adb63ae3-8709-45b4-8f3b-b84877ac2530">

Building block highlighting:

<img width="1727" alt="Screenshot 2024-04-17 at 1 11 13 PM" src="https://github.com/elastic/kibana/assets/17211684/72f708a9-12c9-4240-a99d-3c9930257c08">


To test:

1. Add `xpack.securitySolution.enableExperimental: [unifiedComponentsInTimelineEnabled]` to your `kibana.dev.yml` 
2. Generate test data (endpoint data is fine)
5. Go to the correlations tab and enter this query to see default events/alerts that should have no highlighting
```any where true```
6. Enter this query to see a generic sequence
```
sequence
[any where true]
[any where true]
```
You can also do something like
```
sequence
[any where host.name=={HOST NAME VALUE HERE}]
[any where true]
```
7. You can also create a correlation rule using any of the above queries to generate building block alerts, and then query those alerts in the correlations tab as well
